File: src/com/bean/Customer.java
package com.bean;

public class Customer {
private int userid;
private String password;
public Customer(int userid, String password) {
	super();
	this.userid = userid;
	this.password = password;
}
public int getUserid() {
	return userid;
}
public void setUserid(int userid) {
	this.userid = userid;
}
public String getPassword() {
	return password;
}
public void setPassword(String password) {
	this.password = password;
}
@Override
public String toString() {
	return "Customer [userid=" + userid + ", password=" + password + "]";
}


}

File: src/com/bean/Customer.go
package bean

import (
	"fmt"
)

type Customer struct {
	userid   int
	password string
}

func CreateCustomer(userid int, password string) Customer {
	return Customer{userid: userid, password: password}
}
func (c Customer) getUserid() int {
	return c.userid
}
func (c Customer) setUserid(userid int) {
	c.userid = userid
}
func (c Customer) getPassword() string {
	return c.password
}
func (c Customer) setPassword(password string) {
	c.password = password
}
func (c Customer) String() string {
	return fmt.Sprintf("Customer [userid=%d password=%s]", c.userid, c.password)
}

//@Override
//public String toString() {
//	return "Customer [userid=" + userid + ", password=" + password + "]";
//}
